frostsnapp/native/src/api.rs

```diff
@@ -695,20 +697,20 @@ impl SuperWallet {
     }
 }
 
-pub fn load(app_dir: String) -> anyhow::Result<(Coordinator, Settings)> {
+pub fn load(app_dir: String) -> anyhow::Result<(Coordinator, Settings, BackupManager)> {
```

Comment:
I think we shouldn't have an ever growing tuple here for every bit of app state we need to keep on the rust side. Make a struct that has `Settings` and `BackupManger` inside it. It can also go in the same "context" on the dart side. Probably we should just merge all these top level "contexts" into one `FrostsnapContext`. Coordinator should also be in it rather than be global variable.